stdlib/cache/src/main/ballerina/src/cache/cache_errors.bal

```diff
@@ -16,21 +16,11 @@
 
 import ballerina/log;
 
-# Record type to hold the details of an error.
-#
-# + message - Specific error message of the error.
-# + cause - Any other error, which causes this error.
-public type Detail record {
-    string message;
-    error cause?;
-};
-
-# Represents the reason for the `cache:Error`.
-public const CACHE_ERROR = "{ballerina/cache}Error";
-
 # Represents the Cache error type with details. This will be returned if an error occurred while doing any of the cache
 # operations.
-public type Error error<CACHE_ERROR, Detail>;
+public type CacheError distinct error;
+
+public type Error CacheError;
```

Comment:
Shall we add docs defining the type
`# Represents Cache related errors.`

Reply:
Fixed with dbd25bbdbe9b2ad3f3a7814cffee15eeb82142eb